src/style.css

```diff
@@ -1,3 +1,179 @@
+
 body {
   margin: 0;
 }
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --white: #fff;
+  --lightgrey: #f3f3f3;
+  --darkgrey: #060b35;
+  --grey: #616070;
+  --blue: #00acdc;
+}
+
+.card {
+  width: 200px;
+  height: 406px;
+  flex-shrink: 0;
+  justify-content: center;
+}
+
+.card__rectangle {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+
+  width: 198px;
+  height: 406px;
+  flex-shrink: 0;
+
+  border-radius: 5px;
+  border: 1px solid var(--lightgrey);
+  background: var(--white);
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  flex-shrink: 0;
+  margin: 32px 19px 0;
+}
+
+.card__description {
+  color: var(--darkgrey);
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 18px;
+  margin: 40px 16px 0;
+}
+
+.card__code {
+  color: var(--grey);
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 14px;
+  margin: 4px 82px 0 16px;
+}
+
+.card__price-bloc {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 56px 16px 0;
+}
+
+.card__price-word {
+  display: inline-block;
+
+  color: var(--grey);
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 18px;
+}
+
+.card__price-value {
+  display: inline-block;
+  color: var(--darkgrey);
+  text-align: right;
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 18px;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: var(--blue);
+  width: 166px;
+  height: 40px;
+  flex-shrink: 0;
+  border-radius: 5px;
+  margin: 16px;
+  border: 1px solid var(--blue);
+}
+
+.card__buy {
+  text-decoration: none;
+  font-family: Roboto, sans-serif;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 16px;
+  text-transform: uppercase;
+  color: var(--white);
+  padding: 11px 70px;
+}
+
+.stars {
+  display: flex;
+  position: absolute;
```

Comment:
There is no need to change the flow of the document here
```suggestion

```

Reply:
here I need to use 'position:absolute;', since the block 'star', according to the conditions, must be separate from the block 'card'